server/middlewares/passport/passportCallback.js

```diff
@@ -0,0 +1,38 @@
+/* eslint-disable camelcase */
+/* eslint-disable no-underscore-dangle */
+
+const performCallback = async (accessToken, refreshToken, profile, done) => {
+  const { id, provider } = profile;
+  const { picture, name } = profile._json;
```

Comment:
Why are we disabling camel case and why do we have to use underscore dangle as a variable name.

Reply:
I disabled it because profile returned by Oauth contains properties that are named with an underscore and not camelCase.